streams/src/test/java/org/apache/kafka/streams/kstream/internals/graph/TableSourceNodeTest.java

```diff
@@ -23,41 +23,39 @@
 import org.apache.kafka.streams.kstream.internals.MaterializedInternal;
 import org.apache.kafka.streams.kstream.internals.graph.TableSourceNode.TableSourceNodeBuilder;
 import org.apache.kafka.streams.processor.internals.InternalTopologyBuilder;
-import org.easymock.EasyMock;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import org.powermock.api.easymock.PowerMock;
-import org.powermock.core.classloader.annotations.PrepareForTest;
-import org.powermock.modules.junit4.PowerMockRunner;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
 
-@RunWith(PowerMockRunner.class)
-@PrepareForTest({InternalTopologyBuilder.class})
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
 public class TableSourceNodeTest {
 
     private static final String STORE_NAME = "store-name";
     private static final String TOPIC = "input-topic";
 
-    private final InternalTopologyBuilder topologyBuilder = PowerMock.createNiceMock(InternalTopologyBuilder.class);
+    @Mock private InternalTopologyBuilder topologyBuilder;
 
     @Test
     public void shouldConnectStateStoreToInputTopicIfInputTopicIsUsedAsChangelog() {
         final boolean shouldReuseSourceTopicForChangelog = true;
-        topologyBuilder.connectSourceStoreAndTopic(STORE_NAME, TOPIC);
```

Comment:
You can add 
`doNothing().when(topologyBuilder).connectSourceStoreAndTopic(STORE_NAME, TOPIC);` here.

Reply:
Hi @divijvaidya, tnx for the feedback. I decided to remove `TableSourceNodeTest` from this MR since it's out-of-the scope (Jira ticket says to update only `StreamsMetricsImplTest`).